### PR TITLE
Panacea serves a waiting page until Asclepius is ready

### PR DESCRIPTION
**Pull request by c-brenn**
Until Asclepius is ready, `GET "/"` returns a waiting page. Left it very basic for now, the frontendy folk can prettify it.

### CODE REVIEW

**Comment by c-brenn** [comment]
Rendering without a layout seems to be the nicest way around a few problems.

Namely, meta tags need to be in the `<head>` - using a shared layout would need conditionals. Similarly, the shared layout includes `app.js` - we would need some conditionals around this too.

**Comment by 22a** [comment]
Looks great, I'll centre a loading icon and add some small text. Would you prefer a merge now and make pretty later or vice versa?

**Comment by c-brenn** [comment]
Might as well wait to merge so it doesn't get included in 0.3 without some beautification

File: panacea/web/templates/page/waiting.html.eex [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <title>Panacea Pathways</title>
+    <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+  </head>
+
+  <body>
+    <div class="container">
+      <nav class="navbar navbar-default">
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#">
```
Comment (c-brenn) [inline panacea/web/templates/page/waiting.html.eex]: seeing as the header is shared between the two layouts, it should probably be a shared template. then both layouts can just:
`<%= render "header.html" %>`

**Comment by c-brenn** [comment]
looks good, I am not allowed to review though

**Comment by 22a** [comment]
Some lovely functionality, unfortunately now obsolete thanks to https://github.com/tom-and-the-toothfairies/pathways/pull/77 🦀  time to close, @c-brenn ?

**Comment by c-brenn** [comment]
alas, what could have been. Yeah, we'll give the branch a proper burial after chiron is merged

**Pull request closed without merging**